src/README.md

```diff
@@ -28,18 +43,17 @@ Type `ctrl+space` to open open the IntelliSense suggestions
 
 To simplify syntax and to allow for dynamic parts, like date range filters, the query can contain macros.
 
-Macro example | Description
------------- | -------------
-*$__database* | Will specify the selected database.  This may use the default from the datasource config, or the explicit value from the query editor.
-*$__table* | Will specify the selected database.  This may use the default from the datasource config, or the explicit value from the query editor.
-*$__measure* | Will specify the selected measure.  This may use the default from the datasource config, or the explicit value from the query editor.
-*$__timeFilter* | Will be replaced by an expression that limits the time to the dashboard range
-*$__interval_ms* | Will be replaced by a number that represents the amount of time a single pixel in the graph should cover
-
+| Macro example      | Description                                                                                                                           |
+| ------------------ | ------------------------------------------------------------------------------------------------------------------------------------- |
+| _$\_\_database_    | Will specify the selected database. This may use the default from the datasource config, or the explicit value from the query editor. |
+| _$\_\_table_       | Will specify the selected database. This may use the default from the datasource config, or the explicit value from the query editor. |
+| _$\_\_measure_     | Will specify the selected measure. This may use the default from the datasource config, or the explicit value from the query editor.  |
+| _$\_\_timeFilter_  | Will be replaced by an expression that limits the time to the dashboard range                                                         |
+| _$\_\_interval_ms_ | Will be replaced by a number that represents the amount of time a single pixel in the graph should cover                              |
 
 ## Using Variables in Queries
 
-Grafana template variables can be used in query Timestream query.  See the grafana documentaiton on [template variables](https://grafana.com/docs/grafana/latest/variables/) for more information on how to setup and use variables.
+Grafana template variables can be used in query Timestream query. See the grafana documentaiton on [template variables](https://grafana.com/docs/grafana/latest/variables/) for more information on how to setup and use variables.
```

Comment:
guess we can correct that typo on "documentaiton" while we're at it

Reply:
> "in query Timestream query"

Is that correct?

Reply:
Good catch! 

>Is that correct?

Doesn't look right to me. Copied the variable section from the CloudWatch docs.

Reply:
Maybe it's supposed to be something like "can be used in Timestream queries"

Reply:
Yeah maybe, but with the text copied from CW this should be solved.

Reply:
oops yes, i didn't see it before